lti_consumer/lti_1p1/contrib/django.py

```diff
@@ -0,0 +1,126 @@
+"""
+This module provides functionality for rendering an LTI embed without an XBlock.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+# See comment in docstring for explanation of the usage of ResourceLoader
+from xblockutils.resources import ResourceLoader
+
+from ..consumer import LtiConsumer1p1
+
+
+def lti_embed(
+        *,
+        html_element_id,
+        lti_launch_url,
+        oauth_key,
+        oauth_secret,
+        resource_link_id,
+        user_id,
+        roles,
+        context_id,
+        context_title,
+        context_label,
+        result_sourcedid,
+        person_sourcedid=None,
+        person_contact_email_primary=None,
+        outcome_service_url=None,
+        launch_presentation_locale=None,
+        **custom_parameters
+):
+    """
+    Returns an HTML template with JavaScript that will launch an LTI embed
+
+    IMPORTANT NOTE: This method uses keyword only arguments as described in PEP 3102.
+    Given the large number of arguments for this method, there is a  desire to
+    guarantee that developers using this method know which arguments are being set
+    to which values. This syntax is NOT backwards compatible with python 2.X, but is
```

Comment:
Do you think this should be a breaking change/result in a major version bump for the repo?